cmd/helpers.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+    "github.com/anaskhan96/soup"
+	"os/exec"
+    "runtime"
+)
+
+// fetchHTML - fetches and parses HTML from the given URL
+func fetchHTML(url string) string {
+    doc, err := soup.Get(url)
+    if err != nil {
+        fmt.Println("Error fetching the URL. Make sure you're connected to Amrita WiFi or VPN.")
+        return ""
+    }
+    return doc
+}
+
+
+// openBrowser - opens a URL in the default web browser
+func openBrowser(url string) {
+    var err error
+    switch runtime.GOOS {
+    case "linux":
+        err = exec.Command("xdg-open", url).Start()
+    case "windows":
+        err = exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
+    case "darwin":
+        err = exec.Command("open", url).Start()
+    }
+    if err != nil {
+        log.Fatal(err)
+    }
+}
+
+// Stack implementation using Go slices
```

Comment:
nit: End comments with `full-stop`.
```suggestion
// Stack implementation using Go slices.
```